tasks/build-examples.js

```diff
@@ -16,6 +17,40 @@ var destDir = path.join(__dirname, '..', 'examples');
 var templatesDir = path.join(__dirname, '..', 'config', 'examples');
 
 /**
+ * Returns an array of classes that are explicitly required inside the source
+ * by calling 'goog.require(…)'
+ *
+ * @param {String} src The JavaScript sourcecode to search for goog.require.
+ * @returns {Array} An array of ol-classes that the source goog.requires.
+ */
+function getGoogRequires(src){
+  var googRequires = [];
+  var match = googRequiresRegEx.exec(src);
+  while (match) {
+    googRequires.push(match[1]);
+    match = googRequiresRegEx.exec(src);
+  }
+  return googRequires;
+}
+
+/**
+ * Takes an array of the names of required OpenLayers symbols and returns an
+ * HTML-snippet with an unordered list to the API-docs for the particular
+ * classes.
+ *
+ * @param {String[]} googRequires
+ * @returns {String} The HTML-snippet with the list of links to API-docs.
+ */
+function getLinkToApiHtml(googRequires) {
+  var lis = googRequires.map(function(symb) {
+    var href = 'http://openlayers.org/en/master/apidoc/' + symb + '.html';
```

Comment:
And then there are things like this: http://openlayers.org/en/master/apidoc/ol.html#Coordinate